pkg/cmd/operator/sidecarprobe.go

```diff
@@ -0,0 +1,173 @@
+package operator
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/scylladb/scylla-operator/pkg/controllerhelpers"
+	"github.com/scylladb/scylla-operator/pkg/genericclioptions"
+	"github.com/scylladb/scylla-operator/pkg/naming"
+	"github.com/scylladb/scylla-operator/pkg/signals"
+	"github.com/scylladb/scylla-operator/pkg/version"
+	"github.com/spf13/cobra"
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+	cliflag "k8s.io/component-base/cli/flag"
+	"k8s.io/klog/v2"
+)
+
+type SidecarProbeOptions struct {
+	ListenAddress string
+}
+
+func NewSidecarProbeOptions(streams genericclioptions.IOStreams) *SidecarProbeOptions {
+	return &SidecarProbeOptions{}
+}
+
+func NewSidecarProbeCmd(streams genericclioptions.IOStreams) *cobra.Command {
+	o := NewSidecarProbeOptions(streams)
+
+	cmd := &cobra.Command{
+		Use:   "sidecar-probe",
```

Comment:
it could probably have a more descriptive naming, at this point it could even suggest this probes sidecar's readiness?

Reply:
done